src/components/VersionRestartModal/VersionRestartModal.tsx

```diff
@@ -207,26 +201,25 @@ const VersionTasks: React.VFC<VersionTasksProps> = ({
 
   return buildVariants ? (
     <>
-      <Row>
-        <TaskStatusFilters
-          onChangeBaseStatusFilter={setBaseStatusFilterTerm}
-          onChangeStatusFilter={setVersionStatusFilterTerm}
-          versionId={version?.id}
-          selectedBaseStatuses={baseStatusFilterTerm || []}
-          selectedStatuses={versionStatusFilterTerm || []}
-          filterWidth="50%"
-        />
-      </Row>
-      {buildVariants.map((patchBuildVariant) => (
-        <BuildVariantAccordion
-          versionId={version?.id}
-          key={`accoridan_${patchBuildVariant.variant}`}
-          tasks={patchBuildVariant.tasks}
-          displayName={patchBuildVariant.displayName}
-          selectedTasks={tasks}
-          toggleSelectedTask={toggleSelectedTask}
-        />
-      ))}
+      <TaskStatusFilters
+        onChangeBaseStatusFilter={setBaseStatusFilterTerm}
+        onChangeStatusFilter={setVersionStatusFilterTerm}
+        versionId={version?.id}
+        selectedBaseStatuses={baseStatusFilterTerm || []}
+        selectedStatuses={versionStatusFilterTerm || []}
+      />
+      {[...buildVariants]
```

Comment:
Why are we spreading buildVariants into a new array?

Reply:
the [`sort`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) function sorts arrays in place so if the array is immutable an error will be thrown, and I was seeing the page crash because of it

It would be better to use [`toSorted`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted), but this is a really new function that doesn't have widespread support on browsers yet

Reply:
discussed offline. This is supported by nearly all of our browsers but we might need to upgrade some of our tooling to use this given how new this feature is.